doc/advanced-configuration/outgoing-connections.md

```diff
@@ -1,177 +1,92 @@
-# Outgoing connections
-
-MongooseIM can be configured to talk to external service like databases or HTTP servers in order to get or set the required data.
+MongooseIM can be configured to talk to external services like databases or HTTP servers in order to get or set the required data.
 The interface for outgoing connections management was unified and is now available via the `outgoing_pools` config option for the following type of connections:
 
 * `cassandra` - pool of connections to Cassandra cluster
 * `riak` - pool of connections to Riak cluster
 * `redis` - pool of connections to Redis server
-* `http` - pool of connections to various HTTP(S) servers MongooseIM can talk to, in example HTTP authentication backend or HTTP notifications
+* `http` - pool of connections to various HTTP(S) servers MongooseIM can talk to, for example HTTP authentication backend or HTTP notifications
```

Comment:
This gives me an impression that it's one pool to multiple servers. I'd say `pool of connections to an HTTP(S) server, e.g. an authentication backend or a notification service`